src/app/components/data-preview/data-preview.component.ts

```diff
@@ -13,14 +14,14 @@ interface PreviewOption {
     plugin?: QhanaPlugin,
 }
 
-function isDataApiObject(input: ExperimentDataApiObject | TimelineStepApiObject): input is ExperimentDataApiObject {
+function isDataApiObject(input: ExperimentDataApiObject | TimelineStepApiObject | TimelineSubStepApiObject): input is ExperimentDataApiObject {
     if ((input as any).contentType != null && (input as any).download != null) {
         return true;
     }
     return false;
 }
 
-function isStepApiObject(input: ExperimentDataApiObject | TimelineStepApiObject): input is TimelineStepApiObject {
+function isStepApiObject(input: ExperimentDataApiObject | TimelineStepApiObject | TimelineSubStepApiObject): input is TimelineStepApiObject | TimelineSubStepApiObject {
```

Comment:
Rename to `isStepLikeApiObject` to express that it does not differentiate between steps and substeps (as they are alike enough for this component)